contracts/Staking.sol

```diff
@@ -852,4 +855,16 @@ contract Staking is
             addr := mload(add(bs, 20))
         }
     }
+
+    /// Transfers ownership of talent stake to another address.
+    ///
+    /// @dev This function requires the _talents addresses in array
+    function transferStakes(address[] memory _talents, address _newOwnerAddress) public {
+        for(uint i = 0; i < _talents.length; i++) {
```

Comment:
Is it too expensive to check if there's any stake to be transferred before iterating over the talents list?

I believe that with the current implementation if a user without stakes for some reason call this method with a huge list of talents he will be charged a lot in gas fees.

I know this is unlikely to happen, but if the validation doesn't hurt I believe that we should add it.

Reply:
@RubenSousaDinis in order to check if a stake is already present or not, we'll have to iterate over them to check each, that will mean having two loops, which will be expensive too.

Reply:
I was meaning to check if the value of `stakes[msg.sender]` is present. I believe we can infer if there's at least one stake. Not iterating over each of them.

Reply:
Oh yes, we can do that, keep a check to avoid the loop